cfme/tests/containers/test_relationships_tables.py

```diff
@@ -180,8 +166,7 @@ def test_images_rel(provider, rel):
         obj = Image(name, provider)
 
         val = obj.get_detail('Relationships', rel)
-        if val == '0' or val == 'Unknown image source':
-            continue
+        assert val != 'Unknown image source'
```

Comment:
Before it was allowed to continue if the val was equal to "Unknown image source", now we say it absolutely can't be?